scripts/copy-kubeconfig.yaml

```diff
@@ -0,0 +1,61 @@
+---
+- hosts: kube-master[0]
+  gather_facts: no
+  become: yes
+  tasks:
+  - fetch:
+      src: "/etc/kubernetes/ssl/{{ item }}.pem"
+      dest: "{{ playbook_dir }}/kubectl/{{ item }}.pem"
+      flat: True
+    with_items:
+      - admin-{{ inventory_hostname }}-key
+      - admin-{{ inventory_hostname }}
+      - ca
+  - name: export hostname
+    set_fact:
+      kubectl_name: "{{ inventory_hostname }}"
+
```

Comment:
i would get the actual ip as the hostname may not be a real address, since kargo uses the ip addresses

`kubectl_host: "{{ hostvars[groups['kube-master'][0]].ansible_host }}"`